config/config.yml.example

```diff
@@ -227,6 +232,11 @@ test:
     bucket: 'check-api-test'
     bucket_region: 'us-east-1'
 
+  # Alegre Similarity Thresholds
+  #
+  image_similarity_threshold: 0.7
+  text_similarity_threshold: 0.7
+
```

Comment:
We don't need to repeat the configuration keys in the `test` section if they are the same as the `development` section.